src/Calculator/Views/StateTriggers/CalculatorProgrammerDisplayPanel.xaml

```diff
@@ -1,4 +1,4 @@
-<UserControl x:Class="CalculatorApp.CalculatorProgrammerDisplayPanel"
+﻿<UserControl x:Class="CalculatorApp.CalculatorProgrammerDisplayPanel"
```

Comment:
Why move `CalculatorProgrammerDisplayPanel` into `StateTriggers` folder?

Reply:
`CalculatorProgrammerDisplayPanel` was putting under the wrong place - `StateTriggers` - in the master branch. There're several problems similar to this. Since we don't want to pause this PR for a long time, let's fix it up later.